src/Humanizer/Configuration/OrdinalizerRegistry.cs

```diff
@@ -10,9 +10,9 @@ public OrdinalizerRegistry() : base(new DefaultOrdinalizer())
             Register("en", new EnglishOrdinalizer());
             Register("es", new SpanishOrdinalizer());
             Register("it", new ItalianOrdinalizer());
-            Register("pt-BR", new BrazilianPortugueseOrdinalizer());
```

Comment:
No. pt-PT and pt-BR is the same language, similar to en-US and en-GB. They have some particular differences in some cases, which isn't this case .
When the sub-set of the language is irrelevant we usually use just "pt" or just "en".
To be consistent I've changed the *BrazilianPortugueseOrdinalizer* to *PortugueseOrdinalizer* because the language is Portuguese, there aren't such thing as Brazilian-Portuguese language.